spec/SensioLabs/Behat/PageObjectExtension/PageObject/Factory/CamelcasedClassNameResolverSpec.php

```diff
@@ -77,4 +86,22 @@ function it_throws_an_exception_if_element_class_does_not_exist()
         $this->shouldThrow(new \InvalidArgumentException($expectedMessage))
             ->duringResolveElement('Recent searches');
     }
+
+    function it_resolves_an_element_class_given_as_fqcn()
+    {
+        $this->beConstructedWith(array('\\Foo', '\\Element'));
+
+        $elementFQCN = 'spec\\SensioLabs\\Behat\\PageObjectExtension\\PageObject\\Factory\\Bar';
+        $this->resolvePage($elementFQCN)->shouldReturn($elementFQCN);
+    }
+}
+
+class Foo extends Page
```

Comment:
Ah right, can you use `spec\\SensioLabs\\Behat\\PageObjectExtension\\PageObject\\Factory\\Fixtures\\NamespacedArticleList` and `spec\\SensioLabs\\Behat\\PageObjectExtension\\PageObject\\Factory\\Fixtures\\Element\\NamespacedSearchBox` instead of the new ones? They're used in other examples.